.ci/gitlab-ci.yml

```diff
@@ -0,0 +1,48 @@
+# Gitlab configuraiton for spack/spack
+
+stages:
+  - packages
+
+variables:
+  SPACK_PACKAGES_CHECKOUT_VERSION: refs/pull/342/head
+
+.clone_packages: &clone_packages
+  - mkdir -p ${REPO_DESTINATION}
+  - cd ${REPO_DESTINATION}
+  - git init
+  - git remote add origin https://github.com/spack/spack-packages.git
+  - git fetch --depth 1 origin ${SPACK_PACKAGES_CHECKOUT_VERSION}
+  - git checkout FETCH_HEAD
+  - cd -
```

Comment:
Why not just `git clone --depth 1 --branch ${SPACK_PACKAGES_CHECKOUT_VERSION} https://github.com/spack/spack-packages.git ${REPO_DESTINATION}`